attributes/ssh_publisher.rb

```diff
@@ -37,4 +37,5 @@
 default['ros_buildfarm']['ssh_publisher']['docs_root_dir'] = '/var/repos/docs'
 default['ros_buildfarm']['ssh_publisher']['rosdistro_cache_root_dir'] = '/var/repos/rosdistro_cache'
 default['ros_buildfarm']['ssh_publisher']['status_page_root_dir'] = '/var/repos/status_page'
+default['ros_buildfarm']['ssh_publisher']['ci_archves_root_dir'] = '/var/repos/ci'
```

Comment:
```suggestion
default['ros_buildfarm']['ssh_publisher']['ci_archves_root_dir'] = '/var/repos/ci_archives'
```

Any reason why the path name shouldn' tmatch the configuration name?

Reply:
560e728